source/navigation/types.tsx

```diff
@@ -57,6 +58,10 @@ export type RootStackParamList = {
 	PrintJobRelease: undefined
 }
 
+export type OnChangeTextType = {
+	nativeEvent: {text: React.SetStateAction<string>}
+}
+
```

Comment:
Thoughts on making this an `interface` instead of a named `Object` type?

We don't currently have a standard one way or the other, but I thought I'd check.

Reply:
Actually, looking at how this is used, I wonder if we should split this to to make this more widely-usable?

```typescript
export interface ChangeTextEvent {
    // ...
}
```

```typescript
export interface OnChangeTextHandler {
    onChange: (event: ChangeTextEvent) => void
}
```

Reply:
Sounds perfect, adding this.